akka-projection-jdbc/src/test/scala/akka/projection/jdbc/JdbcProjectionSpec.scala

```diff
@@ -921,8 +921,10 @@ class JdbcProjectionSpec
         concatStr.text shouldBe "abc|def|ghi"
       }
       withClue(s"check: last seen offset is 2L") {
-        val offset = offsetStore.readOffset[Long](projectionId).futureValue.value
-        offset shouldBe 2L
+        eventually {
```

Comment:
That's right, it's not guaranteed that it has saved 2L before processing 3 and 4. There are probably many more of this, also in Cassandra and Slick tests.

Reply:
Yeah, my idea behind the refactor in #284 was to propagate this `eventually` to everywhere.